src/apps/queues/models.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.db import models
+from pyrabbit.http import HTTPError
+
+from queues import rabbit
+
+
+class Queue(models.Model):
+    name = models.CharField(max_length=64)
+    vhost = models.UUIDField(unique=True)
+    is_public = models.BooleanField(default=False)
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.DO_NOTHING,
```

Comment:
I think Eric and I originally thought that we should use `models.DO_NOTHING` because it should still contain a PK reference to the user who made it? (v1.5 was doing something weird in regards to this that we fixed recently, or have a PR open for it.) But setting `null` also makes sense

Reply:
I may be wrong, but _pretty_ sure that postgres won't let you delete a user who still has queue objects pointed to them because of integrity enforcement, so this should be something else?

Reply:
I'll change it to `models.SET_NULL`.

I believe though for that integrity enforcement, we'd have to set `on_delete=models.PROTECT`.
v1.5 issue/PR:
https://github.com/codalab/codalab-competitions/issues/2606
https://github.com/codalab/codalab-competitions/pull/2693

Reply:
Django docs:
![image](https://user-images.githubusercontent.com/37089863/68061262-35063e00-fcc1-11e9-8157-26b5a76fa265.png)

Proven by hand:
![image](https://user-images.githubusercontent.com/37089863/68061254-291a7c00-fcc1-11e9-9aae-f1b292690ed8.png)